diff --git a/nixified-deps/composer-env.nix b/nixified-deps/composer-env.nix
index 6c89bc8..994be52 100644
--- a/nixified-deps/composer-env.nix
+++ b/nixified-deps/composer-env.nix
@@ -188,6 +188,9 @@ let
         # Using the dynamic loader for a Nix package is useless since there is nothing to dynamically reload.
         composer dump-autoload --optimize ${lib.optionalString noDev "--no-dev"} ${composerExtraArgs}
 
+        # some post-intall-cmds may need to modify subdirs of vendor
+        chmod -R u+w vendor
+
         # Run the install step as a validation to confirm that everything works out as expected
         composer install --optimize-autoloader ${lib.optionalString noDev "--no-dev"} ${composerExtraArgs}
 
